aman/src/pages/Home.jsx

```diff
@@ -0,0 +1,40 @@
+import Login from "./Login";
+import background from "../assets/background.jpg";
```

Comment:
Try to use better naming like Background instead of background. Check for other places as well.